lms/templates/vert_module.html

```diff
@@ -6,7 +7,7 @@
 <div class="vert-mod">
 % for idx, item in enumerate(items):
   <div class="vert vert-${idx}" data-id="${item['id']}">
-    ${item['content']}
+    ${item['content'] | n, unicode}
```

Comment:
I've been thinking about these cases and wondering if one could use HTML() when setting item['content']? I can't easily see that code, so that may or may not make sense. If you could, then you wouldn't need the special filter, because this would know not to escape.